plugin/src/main/java/com/denizenscript/denizen/events/block/BlockCooksSmeltsItemScriptEvent.java

```diff
@@ -4,65 +4,69 @@
 import com.denizenscript.denizen.objects.LocationTag;
 import com.denizenscript.denizen.events.BukkitScriptEvent;
 import com.denizenscript.denizencore.objects.ObjectTag;
+import org.bukkit.block.Block;
 import org.bukkit.event.EventHandler;
 import org.bukkit.event.Listener;
-import org.bukkit.event.inventory.FurnaceSmeltEvent;
+import org.bukkit.event.block.BlockCookEvent;
 
-public class FurnaceSmeltsItemScriptEvent extends BukkitScriptEvent implements Listener {
+public class BlockCooksSmeltsItemScriptEvent extends BukkitScriptEvent implements Listener {
 
     // <--[event]
     // @Events
-    // furnace smelts <item> (into <item>)
+    // <block> smelts <item> (into <item>)
+    // <block> cooks <item> (into <item>)

```

Comment:
this should be `cooks|smelts`, exactly the same as the registerCouldMatcher

Reply:
Should be fixed